airbyte-webapp-e2e-tests/cypress/pages/destinationPage.ts

```diff
@@ -1,18 +1,21 @@
-const newDestination = "button[data-id='new-destination'";
-const addSourceButton = "button[data-id='select-source']";
+const newDestination = "button[data-id='new-destination']";
+const addSourceButton = "button[data-testid='select-source']";
 
 export const goToDestinationPage = () => {
   cy.intercept("/api/v1/destinations/list").as("getDestinationsList");
   cy.visit("/destination");
   cy.wait(3000);
-  cy.wait("@getDestinationsList");
 };
 
 export const openNewDestinationForm = () => {
-  cy.get(newDestination).click();
+  cy.wait("@getDestinationsList").then(({ response }) => {
```

Comment:
Since we won’t have a button in the empty state (no destinations set up yet) we need to make a decision based on the response on this request.